### ISSUE TITLE
Rename `id` to `playerId`

### ISSUE THREAD

**Issue by jeremyckahn** [jeremyckahn/farmhand#387]
> I think it would make sense to rename `id` to `playerId` globally, but that's a very wide-ranging change that would be best saved for a focused followup PR.

_Originally posted by @jeremyckahn in https://github.com/jeremyckahn/farmhand/pull/386#discussion_r1090029705_

**Comment by SujayDas1999** [comment]
Hi @jeremyckahn 
If this is not being worked upon yet, I would like to take on this issue :)

**Comment by Henrichy** [comment]
I agree that renaming id to playerId would improve readability and reduce ambiguity. Given the scope of the change, it might be best to address it in a separate, focused PR to keep this one easier to review.

**Comment by jeremyckahn** [comment]
@Henrichy agreed. This actually seems like a tedious task that would be best left to an AI to automate away, so I'll see if I can I do that.

(I forgot about this issue until now! 😅)

**Comment by Henrichy** [comment]
So it's best I leave it right

**Comment by Henrichy** [comment]
I can still assist if you don't mind

**Comment by Henrichy** [comment]
Ok I'll be expecting your feedback
